src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/config/ExceptionHandler.kt

```diff
@@ -82,7 +82,7 @@ class HmppsPrisonerFromNomisMigrationExceptionHandler {
         .body(
           ErrorResponse(
             status = BAD_REQUEST,
-            userMessage = "Bad Request: ${e.cause?.message}",
```

Comment:
This was copied from the ValidationException where it makes sense - doesn't make sense here as the cause is usually null.